core/src/main/java/org/jdbi/v3/core/Jdbi.java

```diff
@@ -224,7 +225,7 @@ public Jdbi installPlugins() {
      * @return this
      */
     public Jdbi installPlugin(JdbiPlugin plugin) {
-        plugin.customizeJdbi(this);
+        Unchecked.runnable(() -> plugin.customizeJdbi(this)).run();
```

Comment:
`Unchecked.consumer(plugin::customizeJdbi).accept(this);` is the right mantra :)